components/shop/hero.module.css

```diff
@@ -0,0 +1,149 @@
+.shopHero {
+  width: 70%;
+  min-height: 64vh;
+  margin: 0 auto 4rem;
+  padding: 4rem 0 0 0;
+  display: flex;
+  flex-flow: wrap row;
+  align-items: stretch;
+  justify-content: center;
+  gap: 2rem;
+}
+
+.shopHero * {
```

Comment:
I'm not a big fan of universal selectors like this, since it used to affect the performance of a page.